app/models/ncr/work_order.rb

```diff
@@ -0,0 +1,42 @@
+module Ncr
+  # Make sure all table names use 'ncr_XXX'
+  def self.table_name_prefix
+    'ncr_'
+  end
```

Comment:
Yep. Without it, active record would look at the `work_order` table. Adding a prefix prevents collisions, but we need to notify active record that that's what's happening.